hard_iron_offset:
  - 1.4275765720571271e-05
  - 7.3093078403483229e-06
  - -5.1347400931062508e-06
soft_iron_matrix:
  -
    - 1
    - 0
    - 0
  -
    - 0
    - 1
    - 0
  -
    - 0
    - 0
    - 1
calibration_quality: 0.95698600024729397
num_samples: 1000